docs/guides/react-components/install-plain-java.md

```diff
@@ -8,8 +8,169 @@ For this installation, you must have:
+      # Controls the number of partitions, which should exist in the cluster.
+      # This can also be overridden using the environment variable ZEEBE_BROKER_CLUSTER_PARTITIONSCOUNT.
+      partitionsCount: 1
+
+    threads:
+      # Controls the number of non-blocking CPU threads to be used.
+      # WARNING: You should never specify a value that is larger than the number of physical cores
+      # available. Good practice is to leave 1-2 cores for ioThreads and the operating
+      # system (it has to run somewhere). For example, when running Zeebe on a machine
+      # which has 4 cores, a good value would be 2.
+      # This setting can also be overridden using the environment variable ZEEBE_BROKER_THREADS_CPUTHREADCOUNT
+      cpuThreadCount: 2
+      # Controls the number of io threads to be used.
+      # This setting can also be overridden using the environment variable ZEEBE_BROKER_THREADS_IOTHREADCOUNT
+      ioThreadCount: 2
+
+    exporters:
+      elasticsearch:
+        className: io.camunda.zeebe.exporter.ElasticsearchExporter
+        args:
+          url: http://localhost:9200
+          index:
+            prefix: zeebe-record
+
+camunda:
+  # Operate configuration properties
+  operate:
+    # Set operate username and password.
+    # If user with <username> does not exists it will be created.
+    # Default: demo/demo
+    #username:
+    #password:
+    # ELS instance to store Operate data
+    elasticsearch:
+      # Cluster name
+      clusterName: elasticsearch
+      # URL
+      url: http://localhost:9200
+    # Zeebe instance
+    zeebe:
+      # Gateway address
+      gatewayAddress: localhost:26500
+    # ELS instance to export Zeebe data to
+    zeebeElasticsearch:
+      # Cluster name
+      clusterName: elasticsearch
+      # URL
+      url: http://localhost:9200
+      # Index prefix, configured in Zeebe Elasticsearch exporter
+      prefix: zeebe-record
+  # Tasklist configuration properties
+  tasklist:
+    # Set Tasklist username and password.
+    # If user with <username> does not exists it will be created.
+    # Default: demo/demo
+    #username:
+    #password:
+    # ELS instance to store Tasklist data
+    elasticsearch:
+      # Cluster name
+      clusterName: elasticsearch
+      # URL
+      url: http://localhost:9200
+    # Zeebe instance
+    zeebe:
+      # Gateway address
+      gatewayAdress: localhost:26500
+    # ELS instance to export Zeebe data to
+    zeebeElasticsearch:
+      # Cluster name
+      clusterName: elasticsearch
+      # Url
+      url: http://localhost:9200
+      # Index prefix, configured in Zeebe Elasticsearch exporter
+      prefix: zeebe-record
+
+```
+
+</details>
+
+:::
+
+Save the file. Without performing this step, no data will be visible in Operate or Tasklist.
+
+3. To start Camunda, run `bin/camunda` (or `bin\camunda.bat` on Windows).
+
+:::tip
+Operate can be found at `http://localhost:8080/` and Tasklist can be found at `http://localhost:8080/tasklist`.
+:::
```

Comment:
I feel like there is a default user/password I can use to log in at this point, but I can't remember what it is or where to find it! We should definitely mention it here.

Reply:
demo/demo